pages/know-your-rights/index.tsx

```diff
@@ -0,0 +1,177 @@
+import Head from 'next/head'
+import React from 'react'
+import { makeStyles, Theme, createStyles } from '@material-ui/core/styles'
+import Card from '@material-ui/core/Card'
+import CardActionArea from '@material-ui/core/CardActionArea'
+import CardActions from '@material-ui/core/CardActions'
```

Comment:
please delete unused imports.

you can run `yarn run lint-es` to help catch these kinds of problems